backend/src/email/middlewares/tests/email.middleware.test.ts

```diff
@@ -21,22 +21,25 @@ beforeEach(() => {
 
 beforeAll(async () => {
   sequelize = await sequelizeLoader(process.env.JEST_WORKER_ID || '1')
-  await User.create({ id: 1, email: 'user@agency.gov.sg' })
+  await User.create({
+    id: 1,
+    email: 'user@agency.gov.sg',
+  } as CreationAttributes<User>)
   const campaign = await Campaign.create({
     name: 'campaign-1',
     userId: 1,
     type: ChannelType.Email,
     valid: false,
     protect: false,
-  })
+  } as CreationAttributes<Campaign>)
   campaignId = campaign.id
 })
 
 afterAll(async () => {
   await Campaign.destroy({ where: {} })
   await User.destroy({ where: {} })
   await sequelize.close()
-  await RedisService.shutdown()
+  // await RedisService.shutdown()
```

Comment:
As you pointed out in our offline discussion, we can remove the commented out lines